master/internal/api_config_policies.go

```diff
@@ -83,16 +83,52 @@ func (a *apiServer) GetWorkspaceConfigPolicies(
 		return nil, err
 	}
 
-	if !configpolicy.ValidWorkloadType(req.WorkloadType) {
-		errMessage := fmt.Sprintf("invalid workload type: %s.", req.WorkloadType)
-		if len(req.WorkloadType) == 0 {
+	resp, err := a.GetConfigPolicies(ctx, ptrs.Ptr(int(req.WorkspaceId)), req.WorkloadType)
+	if err != nil {
+		return nil, err
+	}
+
+	return &apiv1.GetWorkspaceConfigPoliciesResponse{ConfigPolicies: resp}, nil
+}
+
+// Get global task config policies.
+func (a *apiServer) GetGlobalConfigPolicies(
+	ctx context.Context, req *apiv1.GetGlobalConfigPoliciesRequest,
+) (*apiv1.GetGlobalConfigPoliciesResponse, error) {
+	license.RequireLicense("manage config policies")
+
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	permErr, err := cluster.AuthZProvider.Get().CanViewGlobalConfigPolicies(ctx, curUser)
+	if err != nil {
+		return nil, err
+	} else if permErr != nil {
+		return nil, permErr
+	}
+	resp, err := a.GetConfigPolicies(ctx, nil, req.WorkloadType)
+	if err != nil {
+		return nil, err
+	}
+
+	return &apiv1.GetGlobalConfigPoliciesResponse{ConfigPolicies: resp}, nil
+}
+
+func (*apiServer) GetConfigPolicies(
```

Comment:
nit: since this is a helper, can we change this to 
```suggestion
func (*apiServer) getConfigPolicies(
```

Reply:
done